rfcs/endpoints-needed.md

```diff
@@ -0,0 +1,50 @@
+# Enpoints Needed For Each Page
+
+Note all enpoints describe what the endpoint should do not necassarily the name of the endpoint.
+
+### Sign In Page
+
+- Get Username
+- Get Password
+- Check for match (username and password)
+- Directs to Match List Page
+
+  #### Sign Up Page  
+
+  - Check for valid creditials (unique, formatting)
+
+  #### Forgot Password Page
+
+  - Get Username 
+  - Send email with tempory creditials
+
+### Match List Page
+
+- Get all matches (filter by signed in account)
+- Create Match button (sends to create match page)
+- Match Details button (sends to match details page)
+- Get user information (logo, username, team name)
+- Get NFL week 
+
+### Match Details Page
+
+- Add Player Button (Player List Page)
+- Get user information (logo, username, team name)
+- Get player information (points, team name, team logo, team city, position)
+
+### Create Match Page
+
+- Create new Match 
+- Create invitation link
+- Get NFL week
+
+### Player List Page
+
+- Get current team (players)
```

Comment:
Reid left a comment in the other PR to have the `team ID` as a field under the NFL player model. This means when we `GET` players, the current team will be available already